src/page-response-impact/RtSummaryStats.tsx

```diff
@@ -139,6 +141,10 @@ const RtSummaryStats: React.FC<Props> = ({ rtData }) => {
         <LegendContainer>
           <LegendSpan />
           <div>{`Facilities with R(t) < 1.0`}</div>
+          <Spacer x={5} />
+          <HelpButtonWithTooltip placement="top">
+            {`An R(t) < 1.0 means the virus will stop spreading. An R(t) > 1.0 means the virus will spread quickly.`}
```

Comment:
Note: Another acceptable solution:

```suggestion
            An R(t) &lt; 1.0 means the virus will stop spreading. An R(t) &gt; 1.0 means the virus will spread quickly.
```

I think this is better than the one you chose in some aspects and worse in some other aspects. Up to you which to choose!